nahusenayElias/Team_4
# add jobs page

## Related ticket
DRD-54

## In this PR:
- Made a new basic page content called Careers
- Changed the Header component Jobs Navlink to direct to the Jobs page instead of careers.druid.fi

### Testing instructions: 
- git checkout branch
- lando db-import DRD-54.sql.gz
- lando drush cim
- lando drush cr
- remember to change the localhost address of your drupal to frontend/src/services/api.jsx
- Check that Jobs content is loading on the frontend
